src/store/actions/helseSpionActions.ts

```diff
@@ -18,3 +18,41 @@ export const fetchPersonError = (): HelseSpionActionTypes => {
     type: HelseSpionTypes.FETCH_PERSON_ERROR,
   }
 };
+
+export const fetchTokenStarted = (): HelseSpionActionTypes => {
+  return {
+    type: HelseSpionTypes.FETCH_TOKEN_STARTED,
+  }
+};
+
+export const fetchTokenSuccess = (): HelseSpionActionTypes => {
+  return {
+    type: HelseSpionTypes.FETCH_TOKEN_SUCCESS,
+  }
+};
+
+export const fetchTokenError = (): HelseSpionActionTypes => {
+  return {
+    type: HelseSpionTypes.FETCH_TOKEN_ERROR,
```

Comment:
Bra at du håndterer errors. Hva med timeout? Blir det håndtert?

Reply:
Not yet. Skal også have håndtert visning av error bedre.